openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -44,6 +61,13 @@
     color: $copy-staff-color;
   }
 
+  .ui-staff__subtitle {
+    @extend %t-subheading;
+    @include fontSize($f-size-medium);
+    font-weight: 600;
```

Comment:
This doesn't seem to work-- there's something else overriding it with a font-weight of 400. I'll check in what you are requesting, but I will need help getting it to apply.